pkg/cloud/gke/vault/vault_backend.go

```diff
@@ -80,21 +84,35 @@ func CreateGCPServiceAccount(kubeClient kubernetes.Interface, vaultName, namespa
 }
 
 // CreateBucket Creates a bucket in GKE to store the backend (encrypted) data for vault
-func CreateBucket(vaultName, clusterName, projectId, zone string) (string, error) {
+func CreateBucket(vaultName, clusterName, projectID, zone string, recreate bool, batchMode bool, in terminal.FileReader, out terminal.FileWriter, outErr io.Writer) (string, error) {
 	bucketName := BucketName(vaultName)
-	exists, err := gke.BucketExists(projectId, bucketName)
+	exists, err := gke.BucketExists(projectID, bucketName)
 	if err != nil {
 		return "", errors.Wrap(err, "checking if Vault GCS bucket exists")
 	}
 	if exists {
-		return bucketName, nil
+		if !recreate {
+			return bucketName, nil
+		}
+		if batchMode {
```

Comment:
I think, the condition should be `!bachMode` when prompting.

Reply:
thanks - doh - fixed!